addons/payment_test/views/payment_templates.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+<template id="test_payment_checkout" inherit_id="payment.checkout" priority="1">
+            <xpath expr="//t[@t-call='payment.v_check_token_verified']" position="replace">
+                <t t-if="token.acquirer_id.provider=='test'">
+                    <span class="badge badge-success">Test Token</span>
+                </t>
+                <t t-else="" t-call="payment.v_check_token_verified"/>
+            </xpath>
+    </template>
+    <template id="test_payment_manage" inherit_id="payment.manage" priority="1">
+            <xpath expr="//t[@t-call='payment.v_check_token_verified']" position="replace">
+                <t t-if="token.acquirer_id.provider=='test'">
+                    <span class="badge badge-success">Test Token</span>
+                </t>
+                <t t-else="" t-call="payment.v_check_token_verified"/>
+            </xpath>
```

Comment:
This is a very unusual way to change a view behavior. Could you try inheriting the newly created `payment.v_check_token_verified` template instead and change that template's behavior rather than that of the payment form. It makes much more sense IMHO (since you're changing the behavior of the checkmark) and is easier to maintain/debug.